addons/l10n_ch/data/account_vat2011_data.xml

```diff
@@ -77,8 +77,8 @@
         </record>
 
         <record model="account.tax.template" id="vat_25_purchase">
-            <field name="name">TVA 2.5% sur achat B&amp;S (TR)</field>
-            <field name="description">2.5% achat</field>
+            <field name="name">2.5% on goods and services</field>
```

Comment:
VAT missing

Reply:
I added VAT everywhere it could be relative to your proposal